lib/commons/dom/is-visible.js

```diff
@@ -151,7 +151,7 @@ function isVisible(el, screenReader, recursed) {
 
   // detatched virtual node
   if (!el) {
-    const parent = vNode.parent;
+    const parent = vNode && vNode.parent;
```

Comment:
@WilcoFiers I am hitting a case where `vNode` is undefined when `el` was a document fragment (but `el` also got set to `undefined` on line 138 because `el.host` was undefined). So, basically, both `el` and `vNode` are undefined by the time you get to line 154. Based on the comment `// detatched virtual node`, it seems like the assumption here is that this block of code will only run for virtual nodes (which the document fragment is not).

This document fragment appears to have come from a user-agent shadow DOM when we are checking `isVisible` on the SVG that is inside the user-agent shadow DOM:
![image](https://user-images.githubusercontent.com/6416791/142060649-8e161a72-a5f7-4528-9bcd-69405b3350c0.png)

The SVG referenced here is a symbol:
![image](https://user-images.githubusercontent.com/6416791/142060958-3fbbe0b2-5a22-42b7-8394-7d746c1b0ece.png)


Although checking if `vNode` is defined or not would stop the error from happening, maybe we shouldn't get into this situation to begin with. Should SVG element that is in this user-agent shadow DOM be filtered out somehow?